NugetLightRequest.cs

```diff
@@ -1602,13 +1602,10 @@ into newGroup
                     }
                     else
                     {
-                        SemanticVersion latest = new SemanticVersion("0.0.0");
-                        foreach (var p in pkgs) { 
-                            if (p.Version > latest) { 
-                                latest = p.Version;
-                            }
-                        }
-                        pkgs = from p in pkgs where p.Version == latest select p; 
+                        pkgs = from p in pkgs
```

Comment:
Is the orignal implement  "pkgs = from p in pkgs where p.IsLatestVersion select p;" contains the prerelease packages?

Something like below is not enough?
pkgs = from p in pkgs where (p.IsLatestVersion && string.IsNullOrWhiteSpace(p.Version.SpecialVersion)  select p